src/include/btmem.h

```diff
@@ -905,6 +905,15 @@ struct __wt_ref {
 #define WT_REF_SIZE 48
 #endif
 
+/*
+ * A list of WT_REF's.
+ */
+struct __wt_ref_list {
+    WT_REF **list;
+    int entry;     /* next entry available in list */
+    int max_entry; /* how many allocated in list */
+};
```

Comment:
These are at least unsigned, and probably `uint32_t`?

Reply:
Changed to `uint32_t` and later changed into `size_t` with the use of `__wt_realloc_def()` macro as it was producing some warnings.